go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/caddyserver/ingress
 go 1.21
 
 require (
-	github.com/caddyserver/caddy/v2 v2.7.5
+	github.com/caddyserver/caddy/v2 v2.7.6-0.20231024195719-67e567b7ecec
```

Comment:
@Embraser01 Do you think we can wait for the official Caddy release here or there is some specific reason why do we want to bump to an untagged version?

Reply:
Not sure on how long we will have to wait before a tagged version, no reason to update yet, I created this PR after testing it to check if everything was working great.

Maybe @mholt can give us a little bit of insight on when the next version will be released?